Optimisation/Zusammenfassung_Optimisation.tex

```diff
@@ -1051,9 +1051,9 @@ \subsubsection{Algorithm}
 	\begin{equation*}
 		x^1 = x^0 - (A^0)^{-1}\nabla f(x^0)
 	\end{equation*}
-	\item \textbf{Iteration step}: To compute $x^{i+1}$ from ${\color{DodgerBlue1} x^i}$, compute ${\color{DodgerBlue1} \nabla f(x^i)}$, $g^i = {\color{DodgerBlue1} \nabla f(x^i)} - \nabla f(x^{i-1})$ and $d^i = x^i - x^{i-1}$, then
+	\item \textbf{Iteration step}: To compute $x^{i+1}$ from ${\color{DodgerBlue1} x^i}$, compute ${\color{DodgerBlue1} \nabla f(x^i)}$, $g^i = \nabla f(x^i) - \nabla f(x^{i-1})$ and $d^i = x^i - x^{i-1}$, then
```

Comment:
Was intentionally made so that the parts of x^i are denoted, should we do away with the color coding completely?

Reply:
Yes, I think it makes sense to remove the color coding completely. I feel it confuses more than it helps.